sdk/src/main/java/io/coti/sdk/TransactionCreation.java

```diff
@@ -16,6 +18,8 @@
 import java.util.List;
 import java.util.Map;
 
+import static io.coti.sdk.utils.Constants.INSUFFICIENT_TRUST_SCORE_MESSAGE;
+
 public class TransactionCreation {
```

Comment:
i think we should change this to Transaction so that you Transaction t = new Transaction()